src/MizarData/include/MizarData/ActiveFunctionTimePerFrameComparator.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_DATA_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+#define MIZAR_DATA_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+
+#include <utility>
+
+#include "ClientData/ScopeStats.h"
+#include "MizarData/BaselineOrComparison.h"
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+#include "Statistics/Gaussian.h"
+#include "Statistics/StatisticsUtils.h"
+
+namespace orbit_mizar_data {
+
+// TODO(b/236130122) it should be moved to a new library
+// Implements the statistical hypothesis testing procedure aimed to test the equality of total
+// CPU-time of the sampled functions given the sampled rates and the frame track stats.
+// Under the assumption of equality the distribution of the statistic is approximated with normal
+// distribution.
+template <typename Counts, typename FrameTrackStats>
+class ActiveFunctionTimePerFrameComparatorTmpl {
+ public:
+  explicit ActiveFunctionTimePerFrameComparatorTmpl(
+      const Baseline<Counts>& baseline_counts,
+      const Baseline<FrameTrackStats>& baseline_frame_stats,
+      const Comparison<Counts>& comparison_counts,
+      const Comparison<FrameTrackStats>& comparison_frame_stats)
+      : baseline_counts_(baseline_counts),
+        baseline_frame_stats_(baseline_frame_stats),
+        comparison_counts_(comparison_counts),
+        comparison_frame_stats_(comparison_frame_stats) {}
+
+  [[nodiscard]] ComparisonResult Compare(SFID sfid) const {
+    const auto baseline_active_time =
+        ActiveFunctionTime(baseline_counts_, baseline_frame_stats_, sfid);
+    const auto comparison_active_time =
+        ActiveFunctionTime(comparison_counts_, comparison_frame_stats_, sfid);
+    const orbit_statistics::MeanAndVariance non_normalized_stat =
+        NonNormalizedStat(baseline_active_time, comparison_active_time);
+
+    const double stat = non_normalized_stat.mean / std::sqrt(non_normalized_stat.variance);
```

Comment:
what does `stat` stand for?

Reply:
For statistic. In e.g. `ScopeStats` we use its plural.